atlasdb-impl-shared/src/test/java/com/palantir/atlasdb/sweep/queue/TargetedSweeperTest.java

```diff
@@ -122,8 +126,11 @@ public void sweepWithSingleEntryUpdatesMetrics() {
         sweepQueue.sweepNextBatch(ShardAndStrategy.conservative(CONS_SHARD));
 
         TargetedSweepMetricsTest.assertTombstonesPutConservativeEquals(1);
-        TargetedSweepMetricsTest.assertLastSweptConservativeEquals(maxTsForFinePartition(0));
         TargetedSweepMetricsTest.assertSweepTimestampConservativeEquals(getSweepTsCons());
+        TargetedSweepMetricsTest.assertLastSweptTimestampConservativeEquals(maxTsForFinePartition(0));
+
+        punchCurrentTimeAtTimestamp(LOW_TS);
+        TargetedSweepMetricsTest.assertMillisSinceLastSweptConservativeLessThanOneSecond();
```

Comment:
nit: theoretically flaky (if there's a second-long GC pause for some reason). We probably don't need to do anything about it.